scripts/variantstore/wdl/GvsCreateVDS.wdl

```diff
@@ -110,7 +120,14 @@ task create_vds {
         account_name=$(gcloud config list account --format "value(core.account)")
 
         pip3 install --upgrade pip
-        pip3 install hail~{'==' + hail_version}
+
+        if [[ -z "~{hail_wheel}" ]]
+        then
+          pip install ~{hail_wheel}
```

Comment:
this is backwards... `-z` [tests if the string is empty](https://kapeli.com/cheat_sheets/Bash_Test_Operators.docset/Contents/Resources/Documents/index#:~:text=%2Dz-,string%20is%20null,-that%20is%2C%20has), so this `pip` invocation should be swapped with the one currently in the `else`.